From cc5c9e6c1ea1911cb53f1cb0643cbc5f6e4cad1d Mon Sep 17 00:00:00 2001
From: Jim Warner <james.warner@comcast.net>
Date: Sun, 20 May 2018 00:00:00 -0500
Subject: [PATCH 02/65] top: don't mess with groff line length in man document

I've long since forgotten why the attempt to influence
groff line lengths was made. However, I did receive an
email regarding problems formatting postscript output.

Hopefully this patch will eliminate any such problems.

Signed-off-by: Jim Warner <james.warner@comcast.net>
---
 top/top.1 | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/top/top.1 b/top/top.1
index b8405e9..3a00543 100644
--- a/top/top.1
+++ b/top/top.1
@@ -6,11 +6,6 @@
 . This file may be copied under the terms of the GNU Public License.
 ..
 \#  Setup ////////////////////////////////////////////////////////////////
-\#  ** Comment out '.nr' or set to 0 to eliminate WIDTH fiddlin' !
-.nr half_xtra 4
-.
-.ll +(\n[half_xtra] + \n[half_xtra])
-.
 \#                      Commonly used strings (for consistency) ----------
 \#                           - our em-dashes
 .ds Em \fR\ \-\-\ \fR
-- 
2.6.4.windows.1

